packages/wrangler/package.json

```diff
@@ -172,6 +172,7 @@
 		"jest": "^29.7.0",
 		"jest-fetch-mock": "^3.0.3",
 		"jest-websocket-mock": "^2.5.0",
+		"md5-file": "^5.0.0",
```

Comment:
Do we need a dependency for this?

Reply:
We're expecting 5-10GB files so I had a look around for something that had a good implementation for larger files. Would be pretty easy to vendor the particular code path we use, if you'd prefer. For now I'll just pin to the exact version at least.